README.md

```diff
@@ -331,6 +331,7 @@ _See also: [Comparison of CalDAV and CardDAV implementations - Wikipedia](https:
 - [ntfy](https://ntfy.sh/) - Push notifications to phone or desktop using HTTP PUT/POST, with Android app, CLI and web app, similar to Pushover and Gotify. ([Demo](https://ntfy.sh/app), [Source Code](https://github.com/binwiederhier/ntfy), [Clients](https://github.com/binwiederhier/ntfy-android)) `Apache-2.0/GPL-2.0` `Go`
 - [OTS](https://ots.fyi/) - One-Time-Secret sharing platform with a symmetric 256bit AES encryption in the browser. ([Source Code](https://github.com/Luzifer/ots)) `Apache-2.0` `Go`
 - [Papercups](https://papercups.io/) - An open source live customer chat web app written in Elixir. ([Demo](https://app.papercups.io/demo), [Source Code](https://github.com/papercups-io/papercups)) `MIT` `Elixir`
+- [plugNmeet](https://www.plugnmeet.org/) - Scalable, High Performance, Open source web conferencing system. ([Demo](https://demo.plugnmeet.com/login.html), [Source Code](https://github.com/mynaparrot/plugNmeet-server)) `MIT` `Go`
```

Comment:
> https://github.com/mynaparrot/plugNmeet-install#the-components-listed-below-will-be-installed
> docker

> The Language tag is the main server-side requirement for the software

Please change the language/platform tag to `Docker`

Reply:
done

Reply:
Actually installing docker isn't compulsory but having it will be easier to use. Our installer which is optional will install docker but you can use it without docker.

Reply:
Just saw that binaries are available at https://github.com/mynaparrot/plugNmeet-server/releases and there is a very minimal documentation on how to install it without Docker (basically install requirements, and find out the correct setup procedure by reading docker-compose.yml). `Docker/Go` is correct.